pkg/translator/prometheus/normalize_name.go

```diff
@@ -218,7 +222,9 @@ func perUnitMapGetOrDefault(perUnit string) string {
 // Returns whether the slice contains the specified value
 func contains(slice []string, value string) bool {
 	for _, sliceEntry := range slice {
-		if sliceEntry == value {
+		if strings.Contains(sliceEntry, "_") {
```

Comment:
This processing is done only when normalization feature gate is ON, so there is no need to modify this part.